openedx/openedx-platform
# feat: add drf for studio video page

## Description

This PR adds a couple DRFs for the Course Authoring video page. No functionality to the Studio video page has been changed. This change impacts "Developers".

## Testing instructions

1. Test all the functionality on the Studio video page.
2. Confirm that all clicks execute as expected.
3. Navigate to the Course Authoring video page.
4. Check that page loads as expected.

## Deadline

None

## Other information

`course-authoring` PR [#640](https://github.com/openedx/frontend-app-course-authoring/pull/640) is dependent on this PR. `course-authoring` PR #640 should be used used for testing step 3 and 4.
